### PR TITLE
documentation for helpers/mouse.js as specified in #15998

### PR DESCRIPTION
**Pull request by b22jonan**

### CODE REVIEW

**Review by c22abdha — APPROVED** [review]
The documentation seems well-crafted, clear, and consistent with the provided code. It effectively explains the functionalities and parameters involved in each function.

File: DuggaSys/diagram/helpers/mouse.js
Comment (a21liltr) [inline DuggaSys/diagram/helpers/mouse.js:1]: Line 133: More descriptive please. What is 'line'? Object? String? Use correct grammar and full sentences. "If no lines are found *then* an empty array *is returned*.

Comment (a21liltr) [inline DuggaSys/diagram/helpers/mouse.js:1]: Line 186: Misspelling.

**Review by a21liltr — CHANGES_REQUESTED** [review]
See comments.

**Comment by b22jonan** [comment]
both comments have been dealt with

**Review by a21liltr — APPROVED** [review]
lgtm.

**Pull request merged**